activerecord-pg_enum.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "activesupport"
 
   spec.add_development_dependency "appraisal"
-  spec.add_development_dependency "bundler", "~> 1.17"
+  spec.add_development_dependency "bundler", "~> 2.1"
```

Comment:
Bundler 2 is incompatible with my test matrix 🙃

Reply:
Yeah, I could see that CI failed so I was rolling this back to 1.17